### ISSUE TITLE
Error to get telemetry data

### ISSUE THREAD

**Issue by felipedalton** [SCasanova/f1dataR#38]
Hey team! I've been trying to pull telemetry data from the package but I'm getting the following error, could you help me out?

`test <- get_driver_telemetry(2022, 4, driver = 'PER')`

```
Error:
! All columns in a tibble must be vectors.
✖ Column `DriverAhead` is a `numpy.ndarray/python.builtin.object` object.
✖ Column `DistanceToDriverAhead` is a `numpy.ndarray/python.builtin.object` object.
✖ Column `RPM` is a `numpy.ndarray/python.builtin.object` object.
✖ Column `Speed` is a `numpy.ndarray/python.builtin.object` object.
✖ Column `nGear` is a `numpy.ndarray/python.builtin.object` object.
✖ … and 10 more problems.
---
Backtrace:
    ▆
 1. ├─f1dataR::get_driver_telemetry(2022, 4, driver = "PER")
 2. │ └─f1dataR:::py_tel_to_tibble(tel)
 3. │   └─f1dataR:::py_tel_to_tibble(object)
 4. │     └─py_tel_object$tel %>% tibble::as_tibble()
 5. ├─tibble::as_tibble(.)
 6. └─tibble:::as_tibble.data.frame(.)
 7.   └─tibble:::lst_to_tibble(unclass(x), .rows, .name_repair)
 8.     └─tibble:::check_valid_cols(x)
 9.       └─rlang::cnd_signal(...)
> 
```

**Comment by SCasanova** [comment]
Hi! I will look into it very soon

**Comment by pbulsink** [comment]
Hi @felipedalton, are you still having this issue? If so, can you paste the result of calling `sessionInfo()`? Can you also put the output of `reticulate::py_version()`?

I was able to run this ok today on macOS with R4.2.3 and Python 3.9

**Comment by SCasanova** [comment]
Can you confirm if this still happens please @felipedalton

**Comment by pbulsink** [comment]
I think this got worked on in #41 too

**Comment by SCasanova** [comment]
Closed by #41
